spec/models/profile_spec.rb

```diff
@@ -4,11 +4,76 @@
   let(:user) { create(:user, :signed_up) }
   let(:another_user) { create(:user, :signed_up) }
   let(:profile) { create(:profile, user: user) }
-
-  subject { profile }
+  let(:pii) do
```

Comment:
it looks like this is only being used in some tests. What is this project's philosophy on usage of `let` and `before` statements? I generally avoid them altogether but can see how they are helpful when there is setup that is shared for all specs in a file. Do we still use them if the setup is not shared?